src/gpu/intel/concat/simple.hpp

```diff
@@ -93,9 +93,8 @@ struct simple_t : public primitive_t {
                     attr()->has_default_values(), VERBOSE_UNSUPPORTED_ATTR);
             VDISPATCH_CONCAT_SC(set_default_params(), VERBOSE_UNSUPPORTED_TAG);
 
-            VDISPATCH_CONCAT_SC(init_conf(engine),
-                    VERBOSE_PRIMITIVE_CREATION_FAIL,
-                    "reusable_concat init_conf");
+            VDISPATCH_CONCAT_SC(init_conf(engine), VERBOSE_INIT_CONF_FAIL,
+                    "reusable_concat");
```

Comment:
This seems to be inconsistent with the other uses of `VERBOSE_INIT_CONF_FAIL` which use the primitive name, i.e. `concat`. Is there a reason why we're using the primitive name instead of the implementation name for these messages? I think it would make more sense for `VDISPATCH_CONCAT_SC(init_conf(engine), VERBOSE_INIT_CONF_FAIL);` to give the error message "failed to initialize configuration for <impl_name> implementation"

Reply:
That works for me :) As long as each exit point in `init_conf` is covered by a `VDISPATCH` message, I completely agree.